pontoon/base/static/js/fluent_interface.js

```diff
@@ -826,7 +855,7 @@ var Pontoon = (function (my) {
               if (showFTLEditor) {
                 var isRichEditorSupported = self.renderEditor({
                   pk: translated, // An indicator that the string is translated
-                  string: translation,
+                  string: translation
```

Comment:
I'd argue against removing trailing commas, they make diffs better. I've always considered them a good practice in both Python and JS.